config/.sass-lint.yml

```diff
@@ -0,0 +1,125 @@
+# sass-lint config generated by make-sass-lint-config v0.1.2
+#
+# The following scss-lint Linters are not yet supported by sass-lint:
+# ChainedClasses, DisableLinterReason, ElsePlacement, PropertyCount
+# PseudoElement, SelectorDepth, SpaceAroundOperator, TrailingWhitespace
+# UnnecessaryParentReference, Compass::*
+#
+# The following settings/values are unsupported by sass-lint:
+# Linter Comment, option "style"
+# Linter Indentation, option "allow_non_nested_indentation"
+# Linter Indentation, option "character"
+# Linter NestingDepth, option "ignore_parent_selectors"
+# Linter SpaceBeforeBrace, option "allow_single_line_padding"
+
+files:
+  include: '**/*.s+(a|c)ss'
+options:
+  formatter: stylish
+  merge-default-rules: false
+rules:
+  bem-depth: 0
+  border-zero: 0
+  brace-style:
+    - 1
+    - allow-single-line: true
+  class-name-format:
+    - 1
+    - convention: hyphenatedlowercase
+  clean-import-paths:
+    - 1
+    - filename-extension: false
+      leading-underscore: false
+  empty-line-between-blocks:
+    - 1
+    - ignore-single-line-rulesets: true
+  extends-before-declarations: 0
+  extends-before-mixins: 0
+  final-newline:
+    - 1
+    - include: true
+  function-name-format:
+    - 1
+    - allow-leading-underscore: true
+      convention: hyphenatedlowercase
+  hex-length: 0
+  hex-notation:
+    - 0
+    - style: lowercase
+  id-name-format:
+    - 1
+    - convention: hyphenatedlowercase
+  indentation:
+    - 1
+    - size: 2
+  leading-zero: 0
+  mixin-name-format:
+    - 1
+    - allow-leading-underscore: true
+      convention: hyphenatedlowercase
+  mixins-before-declarations: 0
+  nesting-depth:
+    - 1
+    - max-depth: 3
+  no-color-keywords: 1
+  no-color-literals: 1
+  no-css-comments: 1
+  no-debug: 1
+  no-duplicate-properties: 1
+  no-empty-rulesets: 1
+  no-extends: 0
```

Comment:
It would be good to prevent use of extend, can we set this to 1 instead so it'll warn?